packages/runtime/src/client/executor/name-mapper.ts

```
@@ -170,45 +150,30 @@ export class QueryNameMapper extends OperationNodeTransformer {
-            ) {
-                selectAllFromModel =
-                    selection.selection.table?.table.identifier.name ??
-                    this.currentModel;
+            } else if (ReferenceNode.is(selection.selection) && SelectAllNode.is(selection.selection.column)) {
+                selectAllFromModel = selection.selection.table?.table.identifier.name ?? this.currentModel;
                 isSelectAll = true;
             }
 
             if (isSelectAll) {
                 if (!selectAllFromModel) {
                     continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)